js/src/dapps/dappreg/Application/application.css

```diff
@@ -15,10 +15,12 @@
 /* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+@import '../_colors.css';
+
 .body {
   color: #333;
```

Comment:
Probably would have made sense to put this colour in `colors.css` as well. (Minor)